manifests/gnome/gconftool_2.pp

```diff
@@ -39,9 +39,17 @@
     $type_real = $type
   }
 
+  if $::lsbmajdistrelease == '10' {
```

Comment:
Please avoid using lsb* facts. They are only available if the lsb*packages are installed.
Also this if is not picky enough, it will also trigger on other OSfamilies once they reached 10.